src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int THREE = 3;
+    private static final int ZERO = 0;
```

Comment:
constants should have a context meaning of that variable, and answer the questions like what is this number for? Or what does this number represent?
```suggestion
    private static final int MAX_BALLS = 3;
    private static final int START_INDEX = 0;
```